examples/with-material-ui/src/server.js

```diff
@@ -16,18 +14,15 @@ server
   .disable('x-powered-by')
   .use(express.static(process.env.RAZZLE_PUBLIC_DIR))
   .get('/*', (req, res) => {
-    // This is needed in order to deduplicate the injection of CSS in the page.
-    const sheetsManager = new WeakMap();
-    // This is needed in order to inject the critical CSS.
-    const sheetsRegistry = new SheetsRegistry();
+    const sheets = new ServerStyleSheets()
     const markup = renderToString(
-      <JssProvider registry={sheetsRegistry} jss={jss}>
-        <MuiThemeProvider sheetsManager={sheetsManager} theme={theme}>
+      sheets.collect(
+        <MuiThemeProvider theme={theme}>
           <App />
         </MuiThemeProvider>
-      </JssProvider>
+      )
     );
-    const css = sheetsRegistry.toString();
+    const css = sheets.toString();
     res.send(
```

Comment:
You may want to explicitly send a 200. `res.status(200).send(...)`. :)

Reply:
hmm … honestly I wouldn't. But since the official create-razzle-app is doing so, I can change it.